specs/requests/posts_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
```

Comment:
- [ ] You are almost there! However, I was not able to run your tests, there's a duplication error in one of the migration files:

<img width="729" alt="Screen Shot 2022-08-12 at 7 17 29 PM" src="https://user-images.githubusercontent.com/83717285/184459582-d09bdcfd-2dba-4044-b3ce-bf1c41cb8118.png">

It would be great to fix this issue in order to be able to run the tests.

I would recommend moving your `_spec.rb` files into the `spec` folder, but not inside another folder, this sometimes gives you a hard time trying to run the tests.